config/locales/en/steps.yml

```diff
@@ -209,25 +209,15 @@ en:
         edit:
           page_title: Did your client lose their job as a result of being in custody?
           date_job_lost: When did they lose their job?
-      income_before_tax:
-        edit:
-          page_title: Is your client's income currently more than £12,475 a year before tax?
-          income_above_threshold_info: Do not include any employment income if your client has lost their job or ended employment.
-          income_above_threshold: Is your client's income currently more than £12,475 a year before tax?
       frozen_income_savings_assets:
         edit:
           page_title:  Does your client have any income, savings or assets under a restraint or freezing order?
           has_frozen_income_or_assets: Does your client have any income, savings or assets under a restraint or freezing order?
-      client_owns_property:
-        edit:
-          page_title: Does your client own their home, or any other land or property?
-          client_owns_property_info: Include land or property fully or partly owned, inside or outside the UK.
-          client_owns_property: Does your client own their home, or any other land or property?
-      has_savings:
```

Comment:
This key is now missing so you get an I18n error whn you run through the flow and get to the has savings page 

`see this page: /applications/<app id>/steps/income/does_client_have_savings_investments`

on your branch